Doc/library/os.path.rst

```diff
@@ -488,6 +488,41 @@ the :mod:`glob` module.)
       Accepts a :term:`path-like object`.
 
 
+.. function:: splitroot(path)
+
+   Split the pathname *path* into a triad ``(drive, root, tail)`` where:
+
+   1. *drive* is a device name, mount point or the empty string;
+   2. *root* is a sequence of separators following the drive or the empty string; and
+   3. *tail* is anything after the root.
+
+   On POSIX systems, *drive* is always empty. The *root* may be empty (if *path* is
+   relative), a single forward slash (if *path* is absolute), or two forward slashes
+   (implementation-defined per `IEEE Std 1003.1-2017; 4.13 Pathname Resolution
+   <https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap04.html#tag_04_13>`_.)
+   For example::
+
+      >>> splitroot('/home/eryk')
+      ('', '/', 'home/eryk')
+      >>> splitroot('//home/eryk')
+      ('', '//', 'home/eryk')
+      >>> splitroot('///home/eryk')
+      ('', '/', '//home/eryk')
```

Comment:
This is your idea and PR, and an extension of your ongoing work on pathlib. Please credit yourself (e.g. "/home/barney"). You definitely deserve it.